test/ComposableCoW.twap.t.sol

```diff
@@ -322,6 +322,10 @@ contract ComposableCoWTwapTest is BaseComposableCoWTest {
                 GPv2Order.hash(part, settlement.domainSeparator()), signature
             ) == ERC1271.isValidSignature.selector
         );
+
+        // Now remove the order
+        vm.prank(address(safe1));
+        composableCow.remove(keccak256(abi.encode(params)));
```

Comment:
do we want to assert anything afterwards or are we simply fine with it not reverting (seems weird to test an action without checking a post-assumption)?

Reply:
I think from memory there were assertions in another test suite referencing this, but in lieu of that, I've added assertions afterwards to stick to conventions.